client/asset/interface.go

```diff
@@ -558,6 +573,18 @@ type LiveReconfigurer interface {
 	Reconfigure(ctx context.Context, cfg *WalletConfig, currentAddress string) (restartRequired bool, err error)
 }
 
+// Bond is the fidelity bond info generated for a certain account ID, amount,
+// and lock time.
+type Bond struct {
+	Version               uint16
+	AssetID               uint32
+	CoinID                []byte
+	SignedTx, UnsignedTx  []byte
```

Comment:
I would normally expect the `Bond` to have sufficient data to reconstruct the transaction, and vice-versa. If we're using a wallet output, the tx will always be available. If we're using an account-derived privkey, we'll be passing in the key anyway. What is the advantage to passing around these transactions? 

Depending on the answer there, I guess the next question is why have a generalized `SendTransaction` method on every `Wallet` instead of a `SendBond(*Bond)` method on the `Bonder` interface.

Reply:
These go to the server.  It's probably clearer in the comms PR (https://github.com/decred/dcrdex/pull/1819) and the integ PR (https://github.com/decred/dcrdex/pull/1820).

The struct is the output if the method that constructs the transaction, so there's no real need to **re**construct.  This is just all you need to `'postbond'`.